tinyagentos/browser_sessions.py

```
@@ -78,6 +85,11 @@ async def init(self) -> None:
         self.db_path.parent.mkdir(parents=True, exist_ok=True)
         self._db = await aiosqlite.connect(str(self.db_path))
         await self._db.executescript(BROWSER_SESSIONS_SCHEMA)
+        # Best-effort migration for databases created before is_mobile was added.
+        try:
+            await self._db.execute(BROWSER_SESSIONS_MIGRATION)
+        except Exception:
+            pass  # Column already exists
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)